src/foam/u2/wizard/StepWizardController.js

```diff
@@ -299,6 +299,19 @@ foam.CLASS({
           : p.then(() => wizardlet.save()), Promise.resolve());
     },
     async function next() {
+      let wizardlet = this.currentWizardlet;
+
+      // if wizardlet.goNextOnSave if false, simply save the wizardlet and return
+      if ( ! wizardlet.goNextOnSave ) {
```

Comment:
will won't work if the wizardlet with `goNextOnSave` is sandwiched between invisible wizardlets (i.e. we're in the loop below instead). I'm okay with merging this for now though since we only need it in this situation, but add a `// TODO:` comment here or an `issues` axiom in StepWizardController.